charts/airbyte-bootloader/values.yaml

```diff
@@ -4,7 +4,6 @@ global:
   deploymentMode: oss
   secretName: something
   database:
-    secretName: "test-postgresql"
```

Comment:
would it be helpful to leave some placeholder value here?  I'm thinking about if an oss user wanted to use this chart, it might be useful for this key to still be listed in the readme so they know they can set it to whatever (I'm assuming oss users won't have the same secrets naming conventions that we already have).

Reply:
Oh, yes, sure, let me do that
Thank you for bumping this out!

Reply:
Was this a mistake? Can you explain quickly why we are removing this?

Reply:
I'll move it back, I just screwed up logic of looking up the secrets value a little bit(found that only today), let me fix that and push the changes alongside with other folks suggestions and comments

Reply:
Fixed